test/reporter/test-reporter-util.js

```diff
@@ -79,6 +79,7 @@ test('util.sanitizeOutput', function (t) {
   var raw = fs.readFileSync(carriageReturnPath, 'utf-8');
   var expected = fs.readFileSync(carriageReturnExpectedPath, 'utf-8');
   var result = util.sanitizeOutput(raw, '#');
-  t.equals(result, expected, 'there should be a # on every line');
+  result += '\n';
```

Comment:
no the only reason I have added an extra line is because my text editor adds a newline to the end of the fixtures file that I am running against